multichain-testing/package.json

```
@@ -54,7 +54,10 @@
       "**/*.test.ts"
     ],
     "concurrency": 1,
-    "serial": true
+    "serial": true,
+    "environmentVariables": {
+      "LOCKDOWN_STACK_FILTERING": "verbose"
+    }
```

I wondered if we should do this throughout the repo, but I _think_ this accomplishes the same and more:
```suggestion
    "require": [
      "@endo/init/debug.js"
    ],
    "serial": true
```